resources/scponly_user.rb

```diff
@@ -0,0 +1,126 @@
+# true To learn more about Custom Resources, see https://docs.chef.io/custom_resources.html
+
+resource_name :scponly_user
+
+default_action :create
+
+property :write_dir, String, default: 'incoming'
+property :public_key, String
+property :private_key, String
+property :chroot, [true, false], default: true
+property :altroot, String
+property :binaries,
+          Array,
+          default:
+          %w(
+            /bin/chgrp
+            /bin/chmod
+            /bin/chown
+            /bin/ln
+            /bin/ls
+            /bin/mkdir
+            /bin/mv
+            /bin/rm
+            /bin/rmdir
+            /usr/bin/scp
```

Comment:
scponly expects this path
```suggestion
            /bin/scp
```